src/CalculoDV.php

```diff
@@ -156,19 +156,20 @@ public static function banrisulNossoNumero($nossoNumero)
 
     public static function banrisulDuploDigito($campo)
     {
-         $dv1 = Util::modulo10($campo);
-            if ($dv1==9) {
+        $dv1 = Util::modulo10($campo);
+        if ($dv1 > 9)
+            $dv1 = 0;
+
+        $dv2 = Util::modulo11($campo . $dv1, 2, 7, 0, 10);
+        if ($dv2 == 10)
+            $dv1++;
+
+        if ($dv2 == 1)
+            if($dv1==9)
```

Comment:
Não entendi isso, se ele for igual a 1 ele não pode ser igual a 9

Reply:
Se o dígito obtido pelo módulo 10 era igual a 9, considera-se então (9+1=10) DV inválido, neste 
 caso, o DV do módulo 10 automaticamente será igual a 0 e procede-se assim novo cálculo pelo módulo 11.

Reply:
Fiz as alterações baseado nesta sequência: 
1 - Caso o resto obtido no cálculo do módulo 11 seja igual a 1, considera-se o DV inválido.
2 - Soma-se, então, 1 ao DV obtido do módulo 10 e refaz-se o cálculo do módulo 11.
3 - Se o dígito obtido pelo módulo 10 era igual a 9, considera-se então (9+1=10) DV inválido.
4 - Neste caso, o DV do módulo 10 automaticamente será igual a 0 e procede-se assim novo cálculo pelo módulo 11.